theme/oauth-authorize.ftl

```diff
@@ -0,0 +1,111 @@
+[#-- @ftlvariable name="version" type="java.lang.String" --]
+[#import "../_helpers.ftl" as helpers/]
+[#assign PERM_DOMAIN="{ENTER DOMAIN HERE}"]
+
+<!doctype html>
+<html lang="en">
+    [@helpers.permHeadTag title="Reset Password"]
+      <script src="/js/jstz-min-1.0.6.js"></script>
+      [@helpers.captchaScripts showCaptcha=showCaptcha captchaMethod=tenant.captchaConfiguration.captchaMethod siteKey=tenant.captchaConfiguration.siteKey/]
+      <script src="/js/oauth2/Authorize.js?version=${version}"></script>
+      <script src="/js/identityProvider/InProgress.js?version=${version}"></script>
+      [@helpers.alternativeLoginsScript clientId=client_id identityProviders=identityProviders/]
+      <script>
+        Prime.Document.onReady(function() {
+          [#-- This object handles guessing the timezone and filling in the device id of the user --]
+          new FusionAuth.OAuth2.Authorize();
+        });
+      </script>
+    [/@helpers.permHeadTag]
+
+  <body class="permanent">
+    [@helpers.permHeader /]
+    <main>
+      [#-- During a linking work flow, optionally indicate to the user which IdP is being linked. --]
+      [#if devicePendingIdPLink?? || pendingIdPLink??]
+        <p class="mt-0">
+        [#if devicePendingIdPLink?? && pendingIdPLink??]
+          ${theme.message('pending-links-login-to-complete', devicePendingIdPLink.identityProviderName, pendingIdPLink.identityProviderName)}
+        [#elseif devicePendingIdPLink??]
+          ${theme.message('pending-link-login-to-complete', devicePendingIdPLink.identityProviderName)}
+        [#else]
+          ${theme.message('pending-link-login-to-complete', pendingIdPLink.identityProviderName)}
+        [/#if]
+        [#-- A pending link can be cancled. If we also have a device link in progress, this cannot be canceled. --]
+        [#if pendingIdPLink??]
+          [@helpers.link url="" extraParameters="&cancelPendingIdpLink=true"]${theme.message('login-cancel-link')}[/@helpers.link]
+        [/#if]
+        </p>
+      [/#if]
+      <form action="${request.contextPath}/oauth2/authorize" method="POST" class="full">
+        <h1 class="form-title">Authorize Application</h1>
+        [@helpers.oauthHiddenFields/]
+        [@helpers.hidden name="showPasswordField"/]
+        [#if showPasswordField && hasDomainBasedIdentityProviders]
+          [@helpers.hidden name="loginId"/]
+        [/#if]
+
+        [@helpers.prInput type="text" fieldName="loginId" autocomplete="username" placeholder=theme.message('loginId') autofocus=true required=true disabled=(showPasswordField && hasDomainBasedIdentityProviders) /]
+        [#if showPasswordField]
+          [@helpers.prInput type="password" fieldName="password" autocomplete="current-password" autofocus=loginId?has_content placeholder=theme.message('password') /]
+          <!-- [@helpers.captchaBadge showCaptcha=showCaptcha captchaMethod=tenant.captchaConfiguration.captchaMethod siteKey=tenant.captchaConfiguration.siteKey/] -->
```

Comment:
I don't think we're going to turn on captchas here in the near future, so let's remove this:

```suggestion
```